src/v2/components/PhotoGallery/GalleryContent.tsx

```
@@ -0,0 +1,121 @@
+      contentRef,
+      Object.keys(groupedPhotos),
+    );
+    const { refreshPhotos, isRefreshing } = usePhotoSource();
+    const { t } = useI18n();
+    const { refreshIndicatorRef, pullProgress } = usePullToRefresh({
+      onRefresh: refreshPhotos,
+      isRefreshing,
+      containerRef: contentRef,
+    });
+
+    const { errorMessage, finished } = useStartupStage();
```

Biome (1.9.4)

[error] This variable is unused.

Unused variables usually are result of incomplete refactoring, typos and other source of bugs.

(lint/correctness/noUnusedVariables)